src/lint.ts

```diff
@@ -113,6 +118,15 @@ export const createLintCommand = () => {
         .argOptional()
         .default(defaultBranchName),
     )
+    .option(
+      "--compare-to <compare-to>",
+      "the head ref to compare against. Defaults to the current working directory",
+    )
+    .option(
+      "--compare-from <compare-from>",
+      "the base ref to compare against. Defaults to HEAD~1",
+      "HEAD~1",
```

Comment:
I think head of main is a better default than `HEAD~1`. People typically make several broken commits before running lint

Reply:
I updated